data-model/src/main/scala/za/co/absa/enceladus/model/versionedModel/VersionedModel.scala

```diff
@@ -35,6 +35,8 @@ trait VersionedModel extends Exportable {
   val dateDisabled: Option[ZonedDateTime]
   val userDisabled: Option[String]
 
+  val locked: Option[Boolean]
+
```

Comment:
**IMHO important:** I would add `userLocked` and `dateLocked` too.

Reply:
Why, could you elaborate, please?

Reply:
Locking does not change the version. Would be good to know who and when did it.